src/databricks/sql/thrift_backend.py

```diff
@@ -136,6 +139,9 @@ def __init__(
             "_use_arrow_native_timestamps", True
         )
 
+        # Cloud fetch
+        self.max_download_threads = kwargs.get("max_download_threads", 10)
```

Comment:
I did not see a performance improvement from setting max_download_threads to above 10. However, this may differ depending on machine/cluster.